Resources/Prototypes/SS220/Entities/Clothing/Head/BeretNanoTrasenOfficer.yml

```
@@ -0,0 +1,17 @@
+# © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatBeretNanoTrasenOfficer
+  name: Берет Офицера NanoTrasen
+  description: Чёрный офицерский берет с выгравированным щитом и мечом, объявляющий миру, что его владелец является защитником NanoTrasen
```

```suggestion
  description: Чёрный офицерский берет с выгравированным щитом и мечом, объявляющий миру, что его владелец является защитником NanoTrasen.
```
Выгравированным на ткани? хотя может такое и бывает, ладно уж